Doc/library/typing.rst

```diff
@@ -3080,35 +3080,34 @@ Introspection helpers
    Return a dictionary containing type hints for a function, method, module
    or class object.
 
-   This is often the same as ``obj.__annotations__``. In addition,
-   forward references encoded as string literals are handled by evaluating
-   them in ``globals``, ``locals`` and (where applicable)
-   :ref:`type parameter <type-params>` namespaces.
-   For a class ``C``, return
-   a dictionary constructed by merging all the ``__annotations__`` along
-   ``C.__mro__`` in reverse order.
-
-   The function recursively replaces all ``Annotated[T, ...]`` with ``T``,
-   unless ``include_extras`` is set to ``True`` (see :class:`Annotated` for
-   more information). For example:
```

Comment:
I agree with getting rid of the big example and pointing to the `Annotated` docs for in-depth discussion, but I'm not sure we should get rid of this paragraph entirely. Without it, we don't document the `include_extras` parameter here at all, which doesn't seem right.

Reply:
Oh right, I meant to add an entry to the bulleted list about this, but apparently I forgot.